File: Modul-1/soal-10/PRAK110_2410817210017_OrlandoSugian.py
import math
a = 12
c = 5
bkuadrat = a**2 + c**2
b = math.sqrt(bkuadrat)
kll = a + b + c
luas = a * c * 1 / 2
print("Diketahui : ")
print(f"Alas = {c} cm")
print(f"Tinggi = {a} cm")
print()
print()
print("Jawab :")
print(f"Sisi A = {a} cm")
print(f"Sisi B = {b:.0f} cm")
print(f"Sisi C = {c} cm")
print(f"Keliling = {kll:.0f} cm")
print(f"Luas = {luas:.0f} cm")

File: Modul-1/soal-10/PRAK110_2410817210017_OrlandoSugian.c
#include <stdio.h>
#include <math.h>
int main ()
{
    int a = 12, c = 5;
    printf("Diketahui : \n");
    printf("Alas = %d cm\n", c);
    printf("Tinggi = %d cm\n\n", a);
    printf("Jawab : \n");
    printf("Sisi A = %d cm\n", a);
    float bkuadrat = pow (a,2) + pow (c,2);
    float b = sqrt(bkuadrat);
    printf("Sisi B = %.0f cm\n", b);
    printf("Sisi C = %d cm\n", c);
    float kll = a + b + c;
    printf("Keliling = %.0f cm\n", kll);
    float luas = a * c * 1 / 2;
    printf("Luas = %.0f cm\n", luas);
    return 0;


}